nni/compression/pytorch/speedup/compressor.py

```diff
@@ -42,18 +42,23 @@ class ModelSpeedup:
         the index of batch dimension in the dummy_input
     confidence: the confidence coefficient of the sparsity inference. This value is
         actually used as the batchsize of the dummy_input.
+    customized_replace_func: None/Dict
+        If the parameter is not None, then we will use the given function to replace the
+        corresponding modules. The `key` of the dict is the opertor types and the `value`
+        is the replace function of corresponding opertor.
```

Comment:
maybe users don't know how to use it if they didn't see the source code, do we need a simple example to show how to write `customized_replace_func`?

Reply:
good suggestion!

Reply:
good suggestion!

Reply:
need `::` when generating doc.
```
    customized_replace_func: None/Dict
        If the parameter is not None, then we will use the given function to replace the
        corresponding modules. The `key` of the dict is the opertor types and the `value`
        is the replace function of corresponding opertor. The replace function should take
        two input parameters, one is the original module, the second input parameter is tuple
        of the input mask, output mask and weight mask. This replace function should prune the module
        accordingly.
        Here is an example of the replace function(more examples can refer to compress_modules.py)::

            def example_replace(ori_module, masks):
                in_mask, out_mask, weight_mask = masks
                # prune the ori_module to a new smaller module according to the mask
                return new_small_module
```